pom.xml

```diff
@@ -29,7 +29,7 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
         <excluded.generation.package>com/sap/cds/feature/attachments/generated/</excluded.generation.package>
-        <cds.services.version>2.10.6</cds.services.version>
+        <cds.services.version>2.10.7</cds.services.version>
```

Comment:
why not 3.7.2 ?

Reply:
Because we want to ensure compatibility with the LTS Version 2.10.x of CAP Java.

Reply:
we can bump to 3.7.2, but I would prefer to do this in another PR and bump also the version of this plugin to 1.1.0 or 2.0.0 to signal this move.